### PR TITLE
Basic Server Structure

### PR DESCRIPTION
**Pull request by TomOrth**

### CODE REVIEW

**Comment by virtuald** [comment]
You shouldn't commit *.pyc files. Add them to your [.gitignore](https://git-scm.com/docs/gitignore).

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
 # VictiScout-Server
-Server for VictiScout.
+
+Server for VictiScout
+Setup:
+
+1. Create a virtualenv by doing ```virtualenv flask```
```
Comment (virtuald) [inline README.md:6]: If you're going to use virtualenvs, consider using virtualenvwrapper instead. Makes life easier.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+2. Run the environment by doing ```. flask/bin/activate```
+3. Install all the requirements ```pip install -r requirements.txt```
```
Comment (virtuald) [inline README.md:8]: I don't see a `requirements.txt` file

File: models/DBClient.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        #inserts into the collection (table)
+        self.collection.insert(j)
+
+    def getAllResults(self):
+        
+        #gets all scouting data in a cursor
+        cursor = self.collection.find()
+        
+        #converts cursor to json 
+        for doc in cursor:
```
Comment (virtuald) [inline models/DBClient.py:24]: Instead of converting each element to json individually, add all of the docs to the list and _then_ convert that to JSON.

Comment (TomOrth) [reply models/DBClient.py:24]: First time working with pymongo so that is what stackoverflow had reccommended

File: README.md (hunk 2 above)
Comment (TomOrth) [reply README.md:8]: Just checked. I added a requirements.txt

**Comment by TomOrth** [comment]
I got rid of the .pyc files and added .gitignore

**Comment by ErikBoesen** [comment]
@virtuald Okay to merge this?

**Pull request merged**